gammapy/irf/edisp/map.py

```diff
@@ -538,3 +538,34 @@ def peek(self, figsize=(15, 5)):
 
         """
         self.get_edisp_kernel().peek(figsize)
+
+    @classmethod
+    def read(cls, filename, format="gadf", hdu=None, checksum=False):
+        """Read an EDispKernelMap from file and create corresponding object.
+
+        Parameters
+        ----------
+        filename : str or `~pathlib.Path`
+            File name.
+        format : {"gadf", "gtdrm"}, optional
+            File format. Default is "gadf".
+        hdu : str or int
+            HDU location. Default is None.
+        checksum : bool
+            If True checks both DATASUM and CHECKSUM cards in the file headers. Default is False.
+
+        Returns
+        -------
+        edisp : `EDispKernelMap`
+            Energy dispersion kernel map.
+
+        """
+
+        if format == "gadf":
+            return super().read(filename, format, hdu, checksum)
+        elif format == "gtdrm":
```

Comment:
Is there some possibility to verify the checksum in the drm case or is it irrelevant?

Reply:
Maybe a log.info to explicitly write that a single bin EDispKernel map is created?